scss/style.scss

```
@@ -0,0 +1,369 @@
+@import "__fonts.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  width: 100%;
+}
+// header
+.header {
+  &__top {
+    background-image: url(../img/HeaderBackground.png);
+    background-size: cover;
+    min-height: 862px;
+    font-family: "Josefin Sans", sans-serif;
+    color: #ffffff;
+    text-align: center;
+  }
+
+  &__title {
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 52px;
+    text-transform: uppercase;
+    padding-top: 295px;
+  }
+
+  &__subtitle {
+    font-size: 22px;
+    line-height: 28px;
+    letter-spacing: 0.31em;
+    padding-bottom: 50px;
+  }
+  &__form {
+    height: 62px;
+    width: 647px;
+    margin: 0 auto;
+    background: #ffffff;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+  &__input-text {
+    width: 500px;
+    border: none;
+  }
+  &__input-text::placeholder {
+    font-family: "Josefin Sans", sans-serif;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2em;
+    color: #999999;
+    padding: 20px 36px;
+  }
+  &__input-text:focus {
+    border: none;
+  }
+  &__btn {
+    background: #5373fd;
+    color: #ffffff;
+    border: 4px solid #ffffff;
+    padding: 16px 24px;
+    font-family: "Aleo", serif;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+  }
+  &__btn:hover {
+    background: #647ff7;
+  }
```

используй вложенность
```suggestion
    text-transform: uppercase;
    &:hover {
      background: #647ff7;
    }
  }
```